modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -64,7 +64,13 @@
        "NAME" : "Name"
      },
      "ACTIONS": {
-       "CONFIRMATION" : "Confirmation required"
+       "CONFIRMATION" : "Confirmation"
+     },
+     "WARNINGS": {
+       "SERIES_HAS_EVENTS": "This series does contain events. Deleting the series will not delete the events"
```

Comment:
`This series does contain events. Deleting the series will not delete the events*.*`

If you use a full stop, there should also be one at the end.